### PR TITLE
Fix intro to be about Hopsworks

### PR DESCRIPTION
**Pull request by jimdowling**
Cleans up the introduction. New images.

### CODE REVIEW

File: docs/installation_guide/hops-manual-installation-guide.rst [hunk 1]
```diff
@@ -54,12 +54,10 @@ Download the source code for Hops::
...
 
 
-Installing Distributed Database
+Installing the Backend Database
 -------------------------------
 
-Hops uses NDB to store the filesystem metadata. NDB can be install using `Karamel`_. Karamel comes with many sample installation recopies for NDB that can be found in the ``examples`` folder of the Karamel installation. 
-
-Instructions for manually installing NDB is out of the scope of this documentation. We refer you to official `NDB Installation Manual`_ for installing NDB. 
+Hops uses NDB to store the filesystem metadata. NDB can be install using `Karamel`_. Karamel comes with many sample installation recopies for NDB that can be found in the ``examples`` folder of the Karamel installation. Instructions for manually installing NDB is out of the scope of this documentation. We refer you to official `NDB Installation Manual`_ for installing NDB.
```
Comment (tkakantousis) [inline docs/installation_guide/hops-manual-installation-guide.rst]: @jimdowling `installed`

Comment (tkakantousis) [reply docs/installation_guide/hops-manual-installation-guide.rst]: @jimdowling `the official` perhaps?

File: docs/overview/introduction/what-hopsworks.rst [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+  Hopsworks unifies a number of open-source analytics and ML frameworks behind a unified REST API.
+
+
+Hopsworks is a managed platform for scale-out data science, with support for both GPUs and Big Data, in a familiar development environment. Hopsworks can be used either through its User-Interface or via a REST API. Hopsworks unique features are:
+
+* a user-friendly UI for development with the latest open-source platforms for Data Science (Jupyter, Conda, etc),
+* Github-like Projects to manage teams/products/workflows/data,
+* managed GPUs as a Resources - scale out Deep Learning training and hyperparameter optimization,
+* the world's fastest, most-scalable distributed hierarchical filesystem.
```
Comment (tkakantousis) [inline docs/overview/introduction/what-hopsworks.rst]: @jimdowling `filesystem,`

File: docs/overview/introduction/what-hopsworks.rst [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+* data management with HopsFS, Hive, Kafka, and Elasticsearch;
+* training machine learning models on both GPUs and CPUs, including distributed training on GPUs;
+* serving of models in production using Kubernetes, with Hopsworks providing authorized, audited access to scale-out models on TensorFlowServing, SparkML, or ScikitLearn;
+* model management and monitoring with a Spark Streaming application analyzing model usage in near-realtime.
+    
+
+Security
+---------------------
+
+Hopsworks' security model is designed to support the processing of sensitive Datasets in a shared (multi-tenant) cluster. The solution is based on Projects. Within a Project, a use may have one of two different roles, a *Data Owner* - who is like a superuser, and a *Data Scientist* - who is allowed run programs (do analysis), but not allowed to:
```
Comment (tkakantousis) [inline docs/overview/introduction/what-hopsworks.rst]: @jimdowling `a user may have`

Comment (tkakantousis) [inline docs/overview/introduction/what-hopsworks.rst]: @jimdowling `is allowed to run`

File: docs/overview/introduction/what-hopsworks.rst [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+* copy data either in or out of the Project,
+* cross-link the data in the Project with data in other Projects (even if she is a member of the other projects).
```
Comment (tkakantousis) [inline docs/overview/introduction/what-hopsworks.rst]: @jimdowling `even if they are a member`

File: docs/installation_guide/hops-manual-installation-guide.rst (hunk 1 above)
Comment (jimdowling) [reply docs/installation_guide/hops-manual-installation-guide.rst]: done

File: docs/overview/introduction/what-hopsworks.rst (hunk 2 above)
Comment (jimdowling) [reply docs/overview/introduction/what-hopsworks.rst]: done

File: docs/overview/introduction/what-hopsworks.rst (hunk 3 above)
Comment (jimdowling) [reply docs/overview/introduction/what-hopsworks.rst]: done

Comment (jimdowling) [reply docs/overview/introduction/what-hopsworks.rst]: ok

File: docs/overview/introduction/what-hopsworks.rst (hunk 4 above)
Comment (jimdowling) [reply docs/overview/introduction/what-hopsworks.rst]: she is ok here, i think.

**Comment by jimdowling** [comment]
done with fixes.

**Comment by jimdowling** [comment]
I cant write gramatically incorrect prose. Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Theofilos Kakantousis <notifications@github.com> Date: 26/11/2018  16:03  (GMT+01:00) To: logicalclocks/hops-docs <hops-docs@noreply.github.com> Cc: Jim Dowling <jdowling@sics.se>, Mention <mention@noreply.github.com> Subject: Re: [logicalclocks/hops-docs] Fix intro to be about Hopsworks (#54) @tkakantousis commented on this pull request.



In docs/overview/introduction/what-hopsworks.rst:
> +
+Hopsworks provides HopsML as a set of services and platforms to support the full machine learning lifecycle, including:
+
+* data management with HopsFS, Hive, Kafka, and Elasticsearch;
+* training machine learning models on both GPUs and CPUs, including distributed training on GPUs;
+* serving of models in production using Kubernetes, with Hopsworks providing authorized, audited access to scale-out models on TensorFlowServing, SparkML, or ScikitLearn;
+* model management and monitoring with a Spark Streaming application analyzing model usage in near-realtime.
+    
+
+Security
+---------------------
+
+Hopsworks' security model is designed to support the processing of sensitive Datasets in a shared (multi-tenant) cluster. The solution is based on Projects. Within a Project, a use may have one of two different roles, a *Data Owner* - who is like a superuser, and a *Data Scientist* - who is allowed run programs (do analysis), but not allowed to:
+
+* copy data either in or out of the Project,
+* cross-link the data in the Project with data in other Projects (even if she is a member of the other projects).

I'd go with the gender-neutral trend option.

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/logicalclocks/hops-docs","title":"logicalclocks/hops-docs","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/logicalclocks/hops-docs"}},"updates":{"snippets":[{"icon":"PERSON","message":"@tkakantousis commented on #54"}],"action":{"name":"View Pull Request","url":"https://github.com/logicalclocks/hops-docs/pull/54#discussion_r236288904"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/logicalclocks/hops-docs/pull/54#discussion_r236288904",
"url": "https://github.com/logicalclocks/hops-docs/pull/54#discussion_r236288904",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "@tkakantousis commented on 54",
"sections": [
{
"text": "",
"activityTitle": "**Theofilos Kakantousis**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@tkakantousis",
"facts": [

]
}
],
"potentialAction": [
{
"targets": [
{
"os": "default",
"uri": "https://github.com/logicalclocks/hops-docs/pull/54#discussion_r236288904"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 415907359\n}"
}
],
"themeColor": "26292E"
}
]

**Pull request merged**